assets/styles/importer.scss

```diff
@@ -16,6 +16,7 @@
 
 @import 'styles.scss';
 @import "chris";
+@import 'jon.scss';
```

Comment:
Imported partial `jon.scss` should be written as `jon`<br>Prefer double-quoted strings